v0/destinations/freshsales/data/identifyConfig.json

```diff
@@ -0,0 +1,167 @@
+[
+  {
+    "destKey": "emails",
+    "sourceKeys": "email",
+    "sourceFromGenericMap": true,
+    "required": true
+  },
+  {
+    "destKey": "first_name",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "last_name",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "subscription_status",
+    "sourceKeys": [
+      "traits.subscriptionStatus",
+      "context.traits.subscriptionStatus"
+    ]
+  },
+  {
+    "destKey": "job_title",
+    "sourceKeys": [
+      "traits.job_title",
+      "traits.jobTitle",
+      "context.traits.job_title",
+      "context.traits.jobTitle"
+    ]
+  },
+  {
+    "destKey": "work_number",
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "external_id",
+    "sourceKeys": "userId",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "mobile_number",
+    "sourceKeys": ["traits.mobileNumber", "context.traits.mobileNumber"]
+  },
+  {
+    "destKey": "address",
+    "sourceKeys": "address",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "city",
+    "sourceKeys": [
+      "traits.city",
+      "traits.address.city",
+      "context.traits.city",
+      "context.traits.address.city"
+    ]
+  },
+  {
+    "destKey": "state",
+    "sourceKeys": [
+      "traits.state",
+      "traits.address.state",
+      "context.traits.state",
+      "context.traits.address.state"
+    ]
+  },
+  {
+    "destKey": "country",
+    "sourceKeys": [
+      "traits.address.country",
+      "traits.country",
+      "context.traits.address.country",
+      "context.traits.country"
+    ]
+  },
+  {
+    "destKey": "zipcode",
+    "sourceKeys": "zipcode",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "sales_accounts",
+    "sourceKeys": [
+      "traits.salesAccounts",
+      "traits.sales_accounts",
+      "context.traits.salesAccounts",
+      "context.traits.sales_accounts"
+    ]
+  },
+  {
+    "destKey": "territory_id",
```

Comment:
Use metadata to format this to number
`rudder-transformer/v0/util/index.js` Line 745